backend/substrapp/ledger_utils.py

```diff
@@ -119,7 +124,8 @@ def retry_on_error(delay=1, nbtries=15, backoff=2, exceptions=None):
         RpcError,
         LedgerUnavailable,
         LedgerPhantomReadConflictError,
-        LedgerEndorsementPolicyFailure
+        LedgerEndorsementPolicyFailure,
+        LedgerStatusError,
```

Comment:
Could you please explain briefly in the code as a comment or in the PR description why we retry on a `LedgerStatusError` ?

I'm quite sure that we won't remember in a couple of weeks / months why this is required :)

Reply:
Done